presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -326,7 +326,7 @@ public ExtraCredentials()
             checkArgument(!value.isEmpty(), "Credential value is empty");
 
             checkArgument(PRINTABLE_ASCII.matchesAllOf(name), "Credential name contains spaces or is not printable ASCII: %s", name);
-            checkArgument(PRINTABLE_ASCII.matchesAllOf(value), "Credential value contains spaces or is not printable ASCII: %s", name);
+            checkArgument(PRINTABLE_ASCII.matchesAllOf(value), "Credential value contains spaces or is not printable ASCII: %s", value);
```

Comment:
i think `name` was intentional
Is it same to throw exception with credential value? I don't think so.